lib/AST/ASTContext.cpp

```diff
@@ -360,6 +360,7 @@ struct ASTContext::Implementation {
     llvm::DenseMap<std::pair<ProtocolDecl*, Type>, ProtocolType*> ProtocolTypes;
     llvm::FoldingSet<UnboundGenericType> UnboundGenericTypes;
     llvm::FoldingSet<BoundGenericType> BoundGenericTypes;
+    llvm::FoldingSet<BoundGenericType> BoundGenericTypes1;
```

Comment:
Unnecessary?

Reply:
Yeah, sorry for these refactoring leftovers.